solutions/ac-GCDOnBlackboard.mdx

```diff
@@ -76,4 +76,41 @@ int main(){
 
 ```
 </CPPSection>
+	
+<PySection>
+
+```py
+from math import gcd
+n = int(input())
+ans = 0
+arr = list(map(int, input().split()))
+
+prefix_gcd = [0] * n
+prefix_gcd[0] = arr[0]
+
+suffix_gcd = [0] * n
+suffix_gcd[n-1] = arr[n-1]
+
+# prefix_gcd[i] = GCD of a[0], a[1], ..., a[i]
+for i in range(1, n):
+	prefix_gcd[i] = gcd(prefix_gcd[i-1], arr[i])
+
+#suffix_gcd[i] = GCD of a[i], a[i+1], ..., a[n-1]
```

Comment:
put a comment after the `#` for the comment
like `# this is a comment`

Reply:
Could you elaborate more?

Reply:
I think they mean put a space.

Reply:
yeah, so like add a space after the hashtag
```py 
#bad
```
```py
# good
```